ganga/GangaDirac/__init__.py

```diff
@@ -115,12 +115,34 @@ def loadPlugins(config=None):
     from .Lib import Files
 
 def postBootstrapHook():
+    from GangaCore.GPIDev.Lib.Config.Config import getConfig#, config_file_as_text
+    from GangaCore.Runtime.bootstrap import GangaProgram
+    regenerate = False
+
     dirac_conf = getConfig('DIRAC')
     if not dirac_conf['DiracEnvSource'] and not dirac_conf['DiracEnvJSON']:
-        return
+        source = input("Please enter the location of your Dirac UI bashrc file.")
+        if not source:
+            logger.warning("No location specified, Dirac plugin will likely not work!")
```

Comment:
I think this message should explicitly say how this should be set in the `.gangarc` file. Also, would it not be better to make the message a `FATAL` to make ganga abort?

Reply:
I found that I also had to set
```
proxyInitCmd = dirac-proxy-init -g gridpp_user -M
userVO = gridpp
```
in the `[DIRAC]` section.

Reply:
is this because you already had the LHCb package loaded as I don't think I have needed these. I have still to run tests on this branch but I'll check these as well